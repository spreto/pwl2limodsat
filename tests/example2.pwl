c
c Hand generated.
c spreto 05/14/2020
c Piecewise Linear Function pwl file example
c

pwl

b 0 1 0
b 0.25 -1 0
b 0 0 1
b 1 0 -1
b 0.5 -1 0
b 1 -1 0

p 1 1 -2 1 0 1
g 1
g 2
g 3
g 4

p 1 2 0 1 0 1
l 2
g 5
g 3
g 4

p 3 4 -1 2 0 1
l 5
g 6
g 3
g 4
