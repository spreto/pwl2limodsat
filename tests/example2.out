-= Formula phi =-

Unit 1 :: Clause      :: 8 
Unit 2 :: Clause      :: 3 
Unit 3 :: Clause      :: -9 9 
Unit 4 :: Negation    :: 3
Unit 5 :: Implication :: 2 4
Unit 6 :: Negation    :: 5
Unit 7 :: Clause      :: 11 12 
Unit 8 :: Clause      :: 13 
Unit 9 :: Implication :: 7 8
Unit 10 :: Negation    :: 9
Unit 11 :: Minimum     :: 6 10
Unit 12 :: Maximum     :: 1 11
Unit 13 :: Clause      :: 11 12 
Unit 14 :: Clause      :: 13 
Unit 15 :: Implication :: 13 14
Unit 16 :: Negation    :: 15
Unit 17 :: Clause      :: 3 
Unit 18 :: Clause      :: -9 9 
Unit 19 :: Negation    :: 18
Unit 20 :: Implication :: 17 19
Unit 21 :: Negation    :: 20
Unit 22 :: Minimum     :: 16 21
Unit 23 :: Maximum     :: 12 22

-= MODSAT Set Phi =-

-= Linear Piece 1 =-
Formula 1:
Unit 1 :: Clause      :: 3 
Unit 2 :: Clause      :: 3 
Unit 3 :: Negation    :: 2
Unit 4 :: Equivalence :: 1 3
Formula 2:
Unit 1 :: Clause      :: 5 
Unit 2 :: Clause      :: 4 
Unit 3 :: Equivalence :: 1 2
Formula 3:
Unit 1 :: Clause      :: 6 
Unit 2 :: Clause      :: 5 5 
Unit 3 :: Equivalence :: 1 2
Formula 4:
Unit 1 :: Clause      :: 6 
Unit 2 :: Clause      :: 1 
Unit 3 :: Equivalence :: 1 2
Formula 5:
Unit 1 :: Clause      :: 4 
Unit 2 :: Clause      :: 3 
Unit 3 :: Implication :: 1 2
Formula 6:
Unit 1 :: Clause      :: 7 
Unit 2 :: Clause      :: 3 
Unit 3 :: Clause      :: 6 
Unit 4 :: Implication :: 2 3
Unit 5 :: Negation    :: 4
Unit 6 :: Equivalence :: 1 5
Formula 7:
Unit 1 :: Clause      :: 8 
Unit 2 :: Clause      :: 7 7 
Unit 3 :: Equivalence :: 1 2

-= Linear Piece 2 =-

-= Linear Piece 3 =-
Formula 1:
Unit 1 :: Clause      :: 11 
Unit 2 :: Clause      :: 10 
Unit 3 :: Equivalence :: 1 2
Formula 2:
Unit 1 :: Clause      :: 12 
Unit 2 :: Clause      :: 11 11 
Unit 3 :: Equivalence :: 1 2
Formula 3:
Unit 1 :: Clause      :: 10 
Unit 2 :: Clause      :: 11 12 
Unit 3 :: Negation    :: 2
Unit 4 :: Equivalence :: 1 3
Formula 4:
Unit 1 :: Clause      :: 14 
Unit 2 :: Clause      :: 13 
Unit 3 :: Equivalence :: 1 2
Formula 5:
Unit 1 :: Clause      :: 15 
Unit 2 :: Clause      :: 14 14 
Unit 3 :: Equivalence :: 1 2
Formula 6:
Unit 1 :: Clause      :: 15 
Unit 2 :: Clause      :: 1 
Unit 3 :: Equivalence :: 1 2
Formula 7:
Unit 1 :: Clause      :: 13 
Unit 2 :: Clause      :: 3 
Unit 3 :: Implication :: 1 2
